app/models/game.rb

```diff
@@ -0,0 +1,8 @@
+class Game < ActiveRecord::Base
+
+  belongs_to :jedi_user_id, foreign_key: 'sith_user_id', class_name: "User"
+  belongs_to :sith_user_id, foreign_key: 'jedi_user_id', class_name: "User"
```

Comment:
I thought they were suppose to be switched, as the sith_user_id would be like the "opponent" associated with the jedi_user_id. Am I mixed up?